static/js/stream_data.js

```diff
@@ -631,6 +631,12 @@ export function create_streams(streams) {
     }
 }
 
+export function clean_up_description(sub) {
+    if (sub.rendered_description !== undefined) {
+        sub.rendered_description = sub.rendered_description.replace("<p>", "").replace("</p>", "");
+    }
+}
```

Comment:
I don't think I agree with this extraction; my thinking is that this transformation on the `rendered_description` HTML is much better done before saving any new `stream` object we receive on the server, rather than every time we try to render one, since that's a much more robust way to prevent future bugs.

Reply:
(We discussed this over chat.)